src/extension/src/constants.ts

```diff
@@ -205,6 +205,12 @@ export const CONSTANTS = {
   START_PORT: 9502,
   VSCODE_COMMAND: {
     OPEN_FOLDER: "vscode.openFolder"
+  },
+  DEPENDENCY_CHECKER: {
+    NODE: 'node',
+    PYTHON: 'python',
+    PYTHON3: 'python3',
+    PYTHON_LAUNCHER: 'py -3'
```

Comment:
@jasmingoh Why do you need to test with Python Launcher if you are already checking with both `python` and `python3` commands. Are there any cases missed by these two?

Reply:
https://github.com/microsoft/WebTemplateStudio/pull/758#issuecomment-503737686

tldr; `python` and `python3` commands are not consistent when user has both python 2.x and python 3.x installed.